pages/supershardians/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { Box, Button, Container, filter, HStack, SimpleGrid } from "@chakra-ui/react";
+import Card from "components/common/ProfileInfoCard";
```

Comment:
Can we remove the components which are imported but not being used?

![image](https://user-images.githubusercontent.com/7558499/168416366-46e03d69-ef76-4b63-adcf-8976034a69bc.png)

Reply:
Removed